client/src/main/java/io/split/client/SplitClientConfig.java

```diff
@@ -10,11 +10,8 @@
 import pluggable.CustomStorageWrapper;
 
 import java.io.IOException;
-import java.util.HashSet;
-import java.util.LinkedHashSet;
-import java.util.List;
+import java.util.*;
```

Comment:
```suggestion
import java.util.HashSet;
import java.util.LinkedHashSet;
import java.util.List;
import java.util.Properties;
import java.util.concurrent.ThreadFactory;
```

Reply:
I found these imports are required:
import java.util.HashSet;
import java.util.HashMap;
import java.util.Map;
import java.util.Arrays;
import java.util.Set;

Reply:
yes, sure. But you should import each java.util, not use '*'.

Reply:
Yep.